app/ui/browser/views/browser.jsx

```diff
@@ -170,10 +170,6 @@ function attachIPCRendererListeners(browserView) {
   ipcRenderer.on('profile-diff', (_, args) => dispatch(args));
   ipcRenderer.on('focus-url-bar', () => dispatch(ipcActions.focusURLBar()));
   ipcRenderer.on('new-tab', () => dispatch(actions.createTab()));
-
-  // @TODO main process should be sending an id to close a tab
```

Comment:
Correct, but we don't track that information and need to close a specific tab from the main process at this point since we have a better idea of how this thing is architected (i wrote the initial comment FWIW)